kuma/wiki/helpers.py

```diff
@@ -355,3 +355,12 @@ def wiki_url(context, path):
     # finally cache the reversed document URL for a bit
     memcache.set(u'wiki_url:%s:%s' % (locale, path), url, 60 * 5)
     return url
+
+
+@register.function
```

Comment:
I'd rather use a `@register.filter` here since that makes this function be available as something akin to shell pipes, something that is more in line with the template language and the actual thing you're doing in the function. The call would then be: `{{ document.title|addbreaks|safe }}`